docker/wait-for/wait-for.env

```diff
@@ -0,0 +1,7 @@
+# Environment config for wait-for scripts
+
+WAIT_FOR_DB_HOST=mysql
+WAIT_FOR_DB_PORT=3306
```

Comment:
woukld this not be port 3007?

Reply:
I think you mean `3307`, and no, because `3307` is only exposed on the host machine. Internally to docker, the db is still on `3306`.

Reply:
aha, cool thanks